README.md

```diff
@@ -1 +1,2 @@
 # tribut-page
+[Tribut Page](https://KiStanislav.github.io/tribut-page/index.html)
```

Comment:
Your word "tribut" has an error ... Be attentive and check your English.

Reply:
My English are very bad. Corrected!

Reply:
That is not English-problem, it is a lack of attention: the name of repository was correct :)

Reply:
No, I changed the name of the repository